lib/src/main/java/org/exoplatform/social/addons/storage/Utils.java

```diff
@@ -0,0 +1,234 @@
+package org.exoplatform.social.addons.storage;
+
+import java.io.ByteArrayInputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.Map.Entry;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import org.exoplatform.commons.file.model.FileItem;
+import org.exoplatform.commons.utils.ListAccess;
+import org.exoplatform.social.addons.rest.IdentityAvatarRestService;
+import org.exoplatform.social.addons.storage.entity.ConnectionEntity;
+import org.exoplatform.social.addons.storage.entity.IdentityEntity;
+import org.exoplatform.social.addons.storage.entity.ProfileExperienceEntity;
+import org.exoplatform.social.core.identity.model.Identity;
+import org.exoplatform.social.core.identity.model.IdentityWithRelationship;
+import org.exoplatform.social.core.identity.model.Profile;
+import org.exoplatform.social.core.identity.provider.OrganizationIdentityProvider;
+import org.exoplatform.social.core.identity.provider.SpaceIdentityProvider;
+import org.exoplatform.social.core.model.AvatarAttachment;
+import org.exoplatform.social.core.relationship.model.Relationship;
+import org.exoplatform.social.core.service.LinkProvider;
+
+public class Utils {
```

Comment:
Done, thanks. (EntityConverterUtils, Utils is used to know that it's an utility class with static methods)